AdminApi.Sdk/Authentication/XsrfAndApiKeyAuthenticator.cs

```diff
@@ -40,7 +40,7 @@ private async Task<string> GetCookie()
     [MemberNotNull(nameof(_xsrfToken), nameof(_xsrfCookie))]
     private async Task RefreshToken()
     {
-        HttpRequestMessage request = new(HttpMethod.Get, "xsrf");
+        HttpRequestMessage request = new(HttpMethod.Get, "api/v1/xsrf");
```

Comment:
Would be nice to avoid having the Api Version hardcoded in more than 1 place.
We already have the `const` in the `AdminApiEndpoint`, any chance we can either use it or move it elsewhere?

Reply:
I want to get rid of the ApiVersion in the Admin API anyway, because we are and probably will always be the only users of the API. So no need for versioning here.